dbt_subprojects/daily_spellbook/models/evms/evms_info.sql

```diff
@@ -101,5 +101,6 @@ FROM (
         , (21000000, 'corn', 'Corn', 'Layer 1', NULL, 'BTCN', NULL, 'https://cornscan.io/', timestamp '2024-11-19 10:29:11', NULL, NULL, NULL, true)
         , (146, 'sonic', 'Sonic', 'Layer 1', NULL, 'SONIC', NULL, 'https://sonicscan.org/', timestamp '2024-12-01 00:00', NULL, NULL, NULL, true)
         , (57073, 'ink', 'Ink', 'Layer 2', 'Optimistic Rollup', 'ETH', 0x4200000000000000000000000000000000000006, 'https://explorer.inkonchain.com/', timestamp '2024-12-06 15:20', 'OP Stack', 'Ethereum Blobs', 'Ethereum', true)
+        , (2741, 'abstract', 'Abstract', 'Layer 2', 'Optimistic Rollup', 'ETH', 0x4200000000000000000000000000000000000006, 'https://explorer.testnet.abs.xyz/', timestamp '2024-06-17 22:39', 'OP Stack', 'Ethereum Blobs', 'Ethereum', true)
```

Comment:
yeah, i think we wait a little more to be accurate. i don't see much on this token `0x4200000000000000000000000000000000000006` in the block explorer, or any for that matter:
https://abscan.org/tokens

Reply:
If abstract uses `ETH` for gas, then `[0x000000000000000000000000000000000000800a](https://abscan.org/token/0x000000000000000000000000000000000000800a)` may be `ETH`.

Reply:
Updated listed tokens.